libc/syscalls.c

```diff
@@ -263,15 +247,27 @@ long __syscall(long n, long x1, long x2, long x3, long x4, long x5, long x6)
         case SYS_mmap:
             return _syscall_mmap(n, x1, x2, x3, x4, x5, x6);
         default:
+            /* Drop through and let the code below handle the syscall. */
+            break;
+    }
+
+    /* Let liboesyscall handle select system calls. */
```

Comment:
s/handle select system calls/handles selected syscalls

Reply:
Singular is correct.

Reply:
@jhand2 I agree with you, though I think the inconsistencies in existing plurals on line 217 and 273 is creating the confusion. Consider fixing them up.